scripts/core/cmdlist.yml

```diff
@@ -30,6 +30,14 @@ etors:
             command list should be optimized for submission to a single command queue and device engine.
             driver **must** disable any implicit optimizations for distributing work across multiple engines.
             this flag should be used when applications want full control over multi-engine submission and scheduling.
+    - name: IN_ORDER
+      version: "1.7"
+      desc: |
+            commands appended to this command list are executed in-order, with driver implementation
+            enforcing dependencies between them. Application is not required to have the signal event
+            of a given command being the wait event of the next to define an in-order list, and
+            application is allowed to pass signal and wait events to each appended command to implement
+            more complex dependency graphs. Cannot be combined with $X_COMMAND_LIST_FLAG_RELAXED_ORDERING.
```

Comment:
"
and application is allowed to pass signal and wait events to each appended command to implement more complex dependency graphs."

Do we need to mention that?  Event model doesn't change with in order command list , you can still pass events from other sources, normally spec mentions something that change or is not allowed compared to default model, while here we state that something that is already there doesn't change.

Reply:
thanks @MichalMrozek . We dont necessarily need to mention it, correct, but I expect this to be a common question. Basically, with in-order lists, users dont need to enforce dependencies to have an in-order list, but can they get the signal events, as well as define additional dependencies? so this is just to clarify that.